cmd/org/common.go

```diff
@@ -21,8 +21,8 @@ const (
 )
 
 var (
-	awsProfile string = ""
-	output     string = ""
+	awsProfile = ""
+	output     = ""
```

Comment:
Is there a specific reason to remove string here?

Reply:
It's more explicit, so easier to read and already there. I don't see a benefit either way to remove it.